### ISSUE TITLE
bottom of operator buttons disappear when operation performed on iPad 2

### ISSUE THREAD

**Issue by ghost** [phetsims/equality-explorer#135]
<b>Test Device</b>

iPad 2

<b>Operating System</b>

iOS 9.3.5

<b>Browser</b>

Safari 9.0 Mobile

<b>Problem Description</b>

For phetsims/QA/issues/131. If you perform an operation, the bottom of the operator buttons that are not selected can disappear. (See screen capture and video.)

<b>Steps to Reproduce</b>

1. Operations screen.
2. Perform operations until bottom of the operator buttons disappear.

<b>Visuals</b>

![eqexbuttons](https://user-images.githubusercontent.com/31285590/42240623-1c68f0d4-7ec5-11e8-9fbb-b44d3ca75df9.png)

https://drive.google.com/open?id=1qvaDP_sOvLenqusaOmLRCdvop6jpNmup

**Comment by pixelzoom** [comment]
@lmulhall-phet Could you investigate whether this occurs on iOS 10 and 11?

**Comment by ghost** [comment]
It's unique to iOS 9, so far as I can tell.

EDIT: I.e. does not occur on iOS 11.

We don't test on iOS 10 for the reasons outlined [here](https://github.com/phetsims/equality-explorer/issues/134#issuecomment-402833899).

**Comment by pixelzoom** [comment]
I've reproduced this on my iPad2 with iOS 9.3.5.  It happens consistently.

There's nothing sim-specific about this issue, it's likely a browser-specific rendering problem. Since it relatively minor and doesn't affect usability of the sim, my recommendation would be to close this with "will not fix".   If that's not acceptable, then we'll need to have @jonathanolson investigate.

@amanda-phet @ariel-phet Does this need to be addressed, or should we ignore it?

**Comment by jonathanolson** [comment]
Can we check if this happens with `?rootRenderer=canvas`? It looks likely svg-only.

**Comment by ghost** [comment]
I will check `?rootRenderer=canvas` when I get back from class ~12:00.

**Comment by pixelzoom** [comment]
I confirmed that the rendering problem goes away where 1.0.0-dev.125 is run with `?rootRenderer=canvas`.   But then the animation of the operations is slow/jerky.

**Comment by jonathanolson** [comment]
I'll wait on @ariel-phet before looking for workarounds.

**Comment by amanda-phet** [comment]
>There's nothing sim-specific about this issue, it's likely a browser-specific rendering problem. Since it relatively minor and doesn't affect usability of the sim, my recommendation would be to close this with "will not fix".

That's fine with me. If this is a consistent issue though with many sims, and we are still supporting iOS9, it should probably be addressed at some point.

**Comment by pixelzoom** [comment]
Based on @amanda-phet's input (above), I'm going to mark this issue as "deferred", and it will not be addressed for 1.0 release.

**Comment by pixelzoom** [comment]
I discussed workarounds with @jonathanolson on Slack.  Easiest might be to add a transparent Rectangle that is slightly taller than the RadioButtonGroup, something like this:

```js
var transparentRectangle = new Rectangle( 0, 0, operatorControl.width, operatorControl.height + 4, {
  fill: 'transparent',
  center: operatorControl.center
} );
```

**Comment by pixelzoom** [comment]
The workaround in https://github.com/phetsims/equality-explorer/issues/135#issuecomment-404339567 did not work, no change.  Verified that I was indeed testing the workaround by using colors other than `'transparent'`.

**Comment by phet-steele** [comment]
Nobody has said this yet, but isn't this just https://github.com/phetsims/scenery/issues/654? If it is, we should add these screenshots to that issue and probably close this one if we don't find a solution.

**Comment by pixelzoom** [comment]
Thanks for identifying the relevant scenery issue @phet-steele.  I thought this was probably a known scenery issue, which is why I involved @jonathanolson.

**Comment by pixelzoom** [comment]
I considered expanding the bounds of the RadioButtonGroup, but @jonathanolson said "I’m not sure increasing normal bounds would work, since bounds aren’t actually output to SVG at all."   So I'm going to stop looking for a workaround and wait for this to be addressed generally in scenery.

**Comment by pixelzoom** [comment]
Perhaps a workaround like AccordionBox https://github.com/phetsims/graphing-quadratics/issues/86 could be used?

**Comment by pixelzoom** [comment]
According to https://phet.colorado.edu/en/help-center/running-sims, iPad system requirements are:

> iPad: iOS 12+ Safari

This issue was unique to iOS9, which is no longer supported.  Closing.
